pkg/handlers/template.go

```diff
@@ -109,6 +111,35 @@ func (t *Template) DeepCopy() *Template {
 	}
 }
 
+func (l *Template) ExtractDefaults(lowerLang string, r reporeader.RepoReader) (map[string]string, error) {
```

Comment:
This can live off the template for now. Once we plan out the future of repo analysis later on this can move if needed

Reply:
sounds good